### PR TITLE
fix: replace elon with brunocroh (I'm better)

### PR DESCRIPTION
**Pull request by brunocroh**
I think this is a mistake, it's better to recommend my tweets, not Elon

Why merge this:

- I don't believe you need twitter blue to reach an audience, nobody really needs twitter blue
- Elon doesn't post puppies and kittens (I do)
- I'm use nvim/arch btw, Elon probably use Eclipse/Windows
- Elon hack youtube channels to sell crypto scams (I don't)

### CODE REVIEW

**Comment by anurag-roy** [comment]
Bruno for president pls

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala [hunk 1]
```diff
@@ -92,7 +92,7 @@ class RequestQueryFeatureHydrator[
       .add(DDGStatsDemocratsFeature, ddgStatsAuthors.longSeq(Democrats).toSet)
       .add(DDGStatsRepublicansFeature, ddgStatsAuthors.longSeq(Republicans).toSet)
       .add(DDGStatsVitsFeature, ddgStatsAuthors.longSeq(Vits).toSet)
-      .add(DDGStatsElonFeature, ddgStatsAuthors.longValue(Elon))
+      .add(DDGStatsElonFeature, ddgStatsAuthors.longValue(Brunocroh))
```
Comment (roosht3) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala:95]: DDGStatsElonFeature needs to be DDGStatsBrunocrohFeature

Comment (brunocroh) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala]: Thanks for the review, fixed! 🚀 
I believe, we can merge it now.

**Comment by ddanielsantos** [comment]
LGTM

**Comment by isacmoura** [comment]
LGTM

**Comment by danielmelogpi** [comment]
LGTM
This is essential

**Comment by brunosilvadev** [comment]
LGTM

**Comment by navegantes** [comment]
LGTM

**Pull request closed without merging**